csaf/src/report/mod.rs

```diff
@@ -32,11 +32,26 @@ pub struct DocumentKey {
     pub url: String,
 }
 
+impl Default for DocumentKey {
+    fn default() -> Self {
+        Self {
+            distribution_url: Url::parse("https://example.net").unwrap(),
```

Comment:
If the concept of "distribution URL" doesn't work, then we need to find a new way to deal with this. I don't think putting in random values in there is the right approach.